test/qa/conftest.py

```diff
@@ -120,3 +122,59 @@ def _capture_traffic(stop_event):
         process.kill()
     print(f"tshark out {process.stdout.read().strip()[-10:]} - {process.stderr.read().strip()[-10:]}")
     time.sleep(1)
+
+
+@pytest.fixture
+def collect_logs():
+    """Collect logs."""
+    logging.log()
+
+    yield
+
+    logging.log(data=info.collect())
+    logging.log()
+
+
+@pytest.fixture
+def nordvpnd_scope_function(collect_logs):  # noqa: ARG001
+    """Manage the NordVPN daemon start/stop and login/logout states in a function scope."""
+    daemon.start()
+    login.login_as("default")
+
+    yield
+
+    sh.nordvpn.logout("--persist-token")
+    sh.nordvpn.set.defaults()
+    daemon.stop()
+
+
+@pytest.fixture(scope='module')
+def nordvpnd_scope_module():
+    """Manage the NordVPN daemon start/stop and login/logout states in a module scope."""
+    daemon.start()
+    login.login_as("default")
+
+    yield
+
+    sh.nordvpn.logout("--persist-token")
+    daemon.stop()
+
+
+@pytest.fixture(scope='module')
+def add_and_delete_random_route():
+    """Add and delete a random network route."""
+    firewall.add_and_delete_random_route()
+
+
+@pytest.fixture
+def disable_dns_ipv6_and_threat_protection():
+    """Disable DNS over IPv6 and threat protection settings."""
+    lib.set_dns("off")
+    lib.set_ipv6("off")
```

Comment:
Is this still needed after https://github.com/NordSecurity/nordvpn-linux/pull/925/files